client/src/helpers/fixEmail.helper.ts

```diff
@@ -0,0 +1 @@
+export const fixEmail: (txt: string) => string = (text) => text.trim().toLowerCase();
```

Comment:
Can this be renamed to something like "normalizeEmail" or "normalizeString" ?

Reply:
Done - 8a0350d594a9a495cbcd06d0270c664b7e85be9b